content/tutorials/portenta-h7/vs-openmv-ttn/content.md

```diff
@@ -0,0 +1,184 @@
+
+### 1. Setting up the Environment
+
+Start by pointing your browser to www.thethingsnetwork.org and use the Sign Up button to setup an account. Next, then fill all the required fields to complete a new registration (if you already have a TTN account, skip this step and continue by signing in).
+
+![The Things Network homepage](assets/vs_ard_ttn_home.png)
+
+### 2. Creating an App on TTN
+
+Once you have created an account with TTN, you need to create a TTN [application](https://www.thethingsnetwork.org/docs/applications/). An application provides a way to aggregate data from different devices, and then use these data with other 3rd party integrations. Go to your [console](https://console.thethingsnetwork.org), and click on **Applications**
+
+![Select Applications on the Console](assets/vs_ard_ttn_app.png)
+
+Here you'll have a list of all your applications. Now create your first app by pressing the **add application** button.
+
+![Finding the add application button](assets/vs_ard_ttn_new_app.png)
+
+You have now to fill only the first two fields:
+
+- The first one is the **ID** of your app: this must be lowercase and without spaces.
+- The second one is a **Description** of your app, and there's no restrictions on formatting
+
+![Adding a application](assets/vs_ard_ttn_app_param.png)
+
+After completing these two fields, press on the "Add application" button located at the bottom right corner of the page. The dashboard will then show you an overview of the newly created app.
+
+![Adding the App Parameters](assets/vs_ard_ttn_add_app.png)
+
+Let's take a closer look at these sections:
+
+- **Application Overview** and Application EUIS: in order to use this app, you'll need the Application ID and its EUIs. An EUI is a globally unique identifier for networks, gateways applications and devices. The EUIs are used to identify all parts of the LoRaWAN inside the backend server.
+- **Devices**: here you can see and manage all the associated devices (e.g. your Portenta H7 with Vision Shield Lora, Arduino MKR WAN 1300 or MKR WAN 1310), or proceed with the registration of new one.
+- **Collaborators**: here you can see and manage all the app collaborators. To integrate with other collaborative platforms or to manage access rights to the app with other TTN registered profiles.
+- **Access keys**: it's the most sensible information. It is basically the key to gain access to your app, so keep it safe.
+
+### 3. Updating the Modems Firmware
+
+The LoRa module on the Vision Shield can be accessed by using the [MKRWAN library](https://github.com/arduino-libraries/MKRWAN)( if you can't find it in your examples list, you can go to **tools > library manager** and type "MKRWAN library" to install it). This library provides all the APIS to communicate with LoRa and LoRaWAN networks and can be installed from the library manager. The code you need to upload and run is from the **MKRWAN** library, and its name is **MKRWANFWUpdate_standalone**.
+
+//REPLACE
```

Comment:
What's missing here?

Reply:
The //REPLACE is a note to replace the screenshot below.

Reply:
:-) I figured. Can't you just replace it?